.github/workflows/nightly.yml

```diff
@@ -0,0 +1,100 @@
+name: Nightly build
+
+on:
+  schedule:
+    - cron: "0 3 * * *"
+  push:
+    branches:
+      - master
```

Comment:
does it make sense to keep master?

Reply:
Not sure, probably not. Copy-pasta from https://github.com/newrelic/infrastructure-bundle/pull/67.

Reply:
I checked and even if it is not useful we are following with the same approach in all the repos

Reply:
I'll mark it as resolved then.